config/kubernetes/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: coco-guest
```

Comment:
why `coco-guest`?

Reply:
No reason, seemed like an apt name! Do you have any suggestions? It is a coco-stack that runs on the guest side!

Reply:
This guest is easily confused with the guest of the guest VM. `kbs-cluster`, `coco-as` or `coco-tenant` might have clearer meanings.

Reply:
Alright I can call it coco-tenant :-)